lib/simp/beaker_helpers/inspec.rb

```diff
@@ -0,0 +1,169 @@
+module Simp::BeakerHelpers
+
+  # Helpers for working with Inspec
+  class Inspec
+    require 'highline'
```

Comment:
Please move the require down to where it is needed.  Highline, by default, uses global objects which may screw up rubygem-simp-cli spec tests.